forms-flow-admin/src/components/roles/roles.tsx

```diff
@@ -249,7 +249,7 @@ const Roles = React.memo((props: any) => {
   // Delete confirmation
 
   const confirmDelete = () => (
-    <div>
+    <div data-testid="confirm-delete-modal">
```

Comment:
please add the data test id with more specific like role-confirm-delete-modal

Reply:
updated